core/experiments/wp-cli.php

```diff
@@ -40,9 +40,7 @@ public function activate( $args, $assoc_args ) {
 		}
 
 		$experiments_manager = Plugin::instance()->experiments;
-		if ( ! $this->check_experiments_exist( $experiments_manager, $experiments ) ) {
-			\WP_CLI::error( 'Experiments do not exist' . $args[0] );
-		}
+        $this->check_experiments_exist( $experiments_manager, $experiments );
```

Comment:
Indentation

Reply:
Same as above. Indent exists but seems to show weirdly on Github

![image](https://github.com/elementor/elementor/assets/130742176/bb88584f-7c46-4ec8-9c6d-636c6733f497)